core/templates/dev/head/pages/exploration_editor/editor_tab/UnresolvedAnswersOverviewDirective.js

```diff
@@ -150,6 +152,14 @@ oppia.directive('unresolvedAnswersOverview', [
                   // Inject RulesService dynamically.
                   var rulesService = $injector.get(rulesServiceName);
 
+                  var RESPONSE_CONFIRMED_TOAST_MESSAGE = (
+                    'Response of the answer has been confirmed');
+                  var RESPONSE_FIXED_TOAST_MESSAGE = (
+                    'Response of the answer has been fixed');
```

Comment:
`The response for "{{short snippet of answer}}..." has been fixed.`

Reply:
Done